app/views/internal/feedback_messages/_feedback_message.html.erb

```diff
@@ -7,7 +11,7 @@
           Report #<%= feedback_message.id %> - Submitted: <%= time_ago_in_words feedback_message.created_at %> ago
         </a>
         <div class="panel-right-elements">
-          ✉️ Emails Sent: <%= feedback_message_emails.count %> |
+          ✉️ Emails Sent: <%= feedback_message_emails.size %> |
```

Comment:
`count` fails for show pages, but `size` runs either `count` or `length` depending on what works. 👍